components/browser/icons/src/main/java/mozilla/components/browser/icons/BrowserIcons.kt

```diff
@@ -28,21 +39,35 @@ import java.util.concurrent.Executors
  */
 class BrowserIcons(
     private val context: Context,
+    private val httpClient: Client,
```

Comment:
nit: This looks like a breaking change, I think it should be added in the changelog.

```
> Task :samples-browser:processGeckoBetaArmDebugResources
> Task :samples-browser:compileGeckoBetaArmDebugKotlin FAILED
e: /build/android-components/samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt: (53, 56): No value passed for parameter 'httpClient'
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':samples-browser:compileGeckoBetaArmDebugKotlin'.
> Compilation error. See log for more details

```